components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -241,15 +240,8 @@ - (void)displaySnackbarViewForMessage:(MDCSnackbarMessage *)message {
       showSnackbarView:snackbarView
               animated:YES
             completion:^{
-              if ([self isSnackbarTransient:snackbarView]) {
-                snackbarView.accessibilityElementsHidden = YES;
-                UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification,
-                                                message.voiceNotificationText);
-
-              } else {
-                UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification,
-                                                snackbarView);
-              }
+              UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification,
```

Comment:
is there a reason why we removed the `UIAccessibilityAnnouncementNotification` a11y posting for transient snackbars? Wouldn't this change the behavior of snackbar even without setting the shouldEnableAccessibilityViewIsModal flag?

Reply:
You are right. Right now because `UIAccessibilityAnnouncementNotification` is used and `accessibilityElementsHidden` is set to `YES`. No matter `accessibilityViewIsModal` is set or not, the snackbar view itself will not be treated as an accessibility modal. And the announcement will be cut off sometimes if there is another accessibility element to process. So removing this announcement posting is to move focus to the snackbar view, and `accessibilityViewIsModal` is to make sure the overlayView is treated as modal. Please feel free correct me if there is anything not precise.

Reply:
Discussed offline. Addressed in cbac33593203a659ce14762e5f23150161a3865b.